Woodall_Robert/Assignments/ninja_gold/server.py

```diff
@@ -0,0 +1,52 @@
+from flask import Flask, redirect, request, render_template, session
+import random, datetime
+
+app = Flask(__name__)
+app.secret_key = '@#$%#&%**%^%^%^#%^*(*'
+
+@app.route('/')
+def index():
+	if (not 'gold_count' in session) or (not 'activities' in session):
```

Comment:
no need to wrap if conditionals in ()'s unless it alters logic order of operations